python/py-google-auth/Portfile

```diff
@@ -18,14 +18,12 @@ description         simplifies using Google's various server-to-server \
 long_description    {*}${description}
 
 homepage            https://pypi.python.org/pypi/${python.rootname}
-master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
-distname            ${python.rootname}-${version}
 
-checksums           rmd160  077e7c5a61134747ba2879b9fa0710f05615bff4 \
-                    sha256  514e39f4190ca972200ba33876da5a8857c5665f2b4ccc36c8b8ee21228aae80 \
-                    size    121940
+checksums           rmd160  471d535f41aa69f225e62bd74af1779a5fdbed04 \
+                    sha256  ad160fc1ea8f19e331a16a14a79f3d643d813a69534ba9611d2c80dc10439dad \
+                    size    188477
 
-python.versions     27 35 36 37 38 39 310
+python.versions     36 37 38 39 310
```

Comment:
as I said in my previous comment, you can only these subports if there are no dependents. That is not the case here:
```
% port echo depends:py27-google-auth
py27-google-api
py27-google-auth-httplib2
% port echo depends:py35-google-auth
py35-google-api
py35-google-auth-httplib2
```

so you'll need to restrict the version for these two subports to the latest supported one.

Reply:
> restrict the version for these two subports to the latest supported one

How to do this? :-(

Or maybe I should upgrade `py-google-api` and `py-google-auth-httplib2` too, seems no one depends on them:
```
$ port echo depends:py27-google-api
$ port echo depends:py35-google-api
$ port echo depends:py27-google-auth-httplib2
py27-google-api                 
$ port echo depends:py35-google-auth-httplib2
py35-google-api                 
```

Reply:
> > restrict the version for these two subports to the latest supported one
> 
> How to do this? :-(

This is done in many Portfiles so it should be easy enough to find an example. In short, you'd need to do something like this within the ` if {${name} ne ${subport}} {` block:
```
if {${python.version} in "27 35"} {
{
    version             ?.?
    revision            0
    checksums      
```

> Or maybe I should upgrade `py-google-api` and `py-google-auth-httplib2` too, seems no one depends on them:
> 
> ```
> $ port echo depends:py27-google-api
> $ port echo depends:py35-google-api
> $ port echo depends:py27-google-auth-httplib2
> py27-google-api                 
> $ port echo depends:py35-google-auth-httplib2
> py35-google-api                 
> ```

Yep, that would be a good solution - we're anyway trying to remove EOL Python versions.